lua/sh-tas-utils/copymodelproperty.lua

```diff
@@ -0,0 +1,18 @@
+
+AddCSLuaFile()
+
+properties.Add("copy_model", {
+	MenuLabel = "Copy Model",
+	Order = 2137,
```

Comment:
Seems like an oddly specific number, but looking at the docs it doesn't matter much anyway

Reply:
Meme number.